pages/linux/vnstati.md

```diff
@@ -0,0 +1,20 @@
+# vnstati
+
+> PNG image output support for vnStat.
+> More information: <https://manned.org/vnstati>.
+
+- Output a summary of the last 2: months, days, and all-time:
```

Comment:
`last 2`:  Why? Is there a default value?

Reply:
What does is mean?

Is is something like:
> Output a summary of the last 2 months, last 2 days, and all-time:

Reply:
> 
> 
> What does is mean?
> 
> Is it something like:
> 
> > Output a summary of the last 2 months, last 2 days, and all-time:

It shows the summary of today, yesterday, the current month, the previous month and the total usage. [This is the output.](https://i.imgur.com/exDJ0WI.png)